BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -51,6 +83,29 @@ public string CheckPortableExecutable(string file, PortableExecutable pex, bool
             if (stxt371Section || stxt774Section)
                 return $"SafeDisc {Get320to4xVersion(null, null, null)}";
 
+            // Get the BSS section, if it exists
+            var BSSSectionRaw = pex.ReadRawSection("BSS", first: true);
```

Comment:
Please follow naming convention: `BSSSectionRaw` => `bssSectionRaw` even if the section name is `BSS`.

Reply:
Does this also apply to normal `.bss` sections as well?